docs/quickstart/fan_out_replication.rst

```diff
@@ -0,0 +1,99 @@
+.. include:: ../global.rst.inc
+
+.. _quickstart-fan-out-replication:
+
+Fan-out replication
+===================
+
+This quick-start example demonstrates how to implement a fan-out replication setup where datasets on a server (A) are replicated to multiple targets (B, C, etc.).
+
+This example uses multiple ``source`` jobs on server A and ``pull`` jobs on the target servers.
+
+Caveats
+-------
+
+Note that it is currently not possible to use a single ``source`` job with multiple ``pull`` jobs. 
+The way zfs abstractions are named, which can be seen in ``zrepl zfs-abstraction list``, include the name of the ``source`` job, but not the name of the client identity of the target servers (see :issue:`380`).
+Multiple ``pull`` jobs with one ``source`` job would share the same zfs replication cursor bookmark and potentially break incremental replication.
+Thus, each client with a ``pull`` job must have its own corresponding ``source`` job.
+
+Since there are multiple ``source`` jobs referencing the same datasets, snapshotting and pruning should be done by a separate ``snap`` job.
```

Comment:
Do you think this section would be better placed in `docs/configuration/overview.rst`? We should definitely link to it from here, though. Probably from within a `.. WARN:` box